clang/lib/Serialization/ASTReaderStmt.cpp

```diff
@@ -2135,9 +2135,14 @@ void ASTStmtReader::VisitUnresolvedLookupExpr(UnresolvedLookupExpr *E) {
 
 void ASTStmtReader::VisitTypeTraitExpr(TypeTraitExpr *E) {
   VisitExpr(E);
+  E->TypeTraitExprBits.IsBooleanTypeTrait = Record.readInt();
```

Comment:
```suggestion
  E->TypeTraitExprBits.IsBooleanTypeTrait = Record.readBool();
```

??

Reply:
This is not a thing! `readInt()` is basically any integer like

Reply:
Is too :-P https://clang.llvm.org/doxygen/classclang_1_1ASTRecordReader.html#a060c24731d3fcb4aa28d74f191cedccd